js/libs/form.js

```diff
@@ -822,6 +822,12 @@ Fliplet.Widget.instance('form-builder', function(data) {
                   throw new Error('Options must be an array');
                 }
 
+                if (field._type === 'flSelect') {
+                  _.remove(values, function(val) {
+                    return val === null  || val === undefined || val.trim() === '';
```

Comment:
there's a double space here which can be removed

Reply:
fixed